src/unit_frame.cpp

```diff
@@ -649,10 +649,8 @@ void unit_frame::redraw(const int frame_time,bool on_start_time,bool in_scope_of
 			sound::play_sound(current_data.sound);
 		}
 		if(!current_data.text.empty()  ) {
-			game_display::get_singleton()->float_label(src,current_data.text,
-			(current_data.text_color & 0x00FF0000) >> 16,
-			(current_data.text_color & 0x0000FF00) >> 8,
-			(current_data.text_color & 0x000000FF) >> 0);
+			game_display::get_singleton()->float_label(src, current_data.text,
```

Comment:
He's using the exact same colour that was used originally; the only concern is whether `int_to_color` interprets it in the same way as the removed code here interprets it, that is, as ARGB. (Or maybe that would be BGRA... endianness is confusing.)